### ISSUE TITLE
Bruno crashes / white screens after running simple GET API request

### ISSUE THREAD

**Issue by Ma-ve** [usebruno/bruno#279]
**Title**
Am unable to run a specific API call in Bruno, instant white screen

**Version**

```
Bruno v0.16.6
Electron 21.1.1
Chrome 106.0.5249.103
Node 16.16.0
v8 10.6.194.17-electron.0
```

**Description**

After hitting 'Send' it'll throw the following error:

```
TypeError: Cannot add property modeOption, object is not extensible
    at new Bs (_app-bf8c7df48b19b842.js:1:193175)
    at Bs (_app-bf8c7df48b19b842.js:1:193021)
    at n.value (index-20bee48e3499f8e1.js:1:45246)
    at ki (framework-49f1e091cbf6b261.js:1:101099)
    at bi (framework-49f1e091cbf6b261.js:1:100872)
    at yi (framework-49f1e091cbf6b261.js:1:100408)
    at framework-49f1e091cbf6b261.js:1:112245
    at ws (framework-49f1e091cbf6b261.js:1:112760)
    at ls (framework-49f1e091cbf6b261.js:1:105218)
    at S (framework-49f1e091cbf6b261.js:1:138465)
```

with the following stacktrace:

```
    at n (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/index-20bee48e3499f8e1.js:1:44739)
    at div
    at div
    at M (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:986973)
    at Lt (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/index-20bee48e3499f8e1.js:1:78052)
    at section
    at div
    at M (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:986973)
    at cn (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/index-20bee48e3499f8e1.js:1:88748)
    at section
    at section
    at div
    at M (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:986973)
    at or (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/index-20bee48e3499f8e1.js:1:125438)
    at section
    at div
    at M (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:986973)
    at div
    at io (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/index-20bee48e3499f8e1.js:1:165192)
    at main
    at div
    at ao
    at div
    at kr (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:463742)
    at d (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:467917)
    at div
    at M (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:986973)
    at ae (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:424564)
    at div
    at fe (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:425601)
    at Te (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:985062)
    at p (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:515223)
    at l (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:947808)
    at Pr (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:467084)
    at div
    at Tr (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:466987)
    at r (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:387342)
    at Rr (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/pages/_app-bf8c7df48b19b842.js:1:467199)
    at r (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/main-ad5649da2d8f61d3.js:1:6769)
    at Y (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/main-ad5649da2d8f61d3.js:1:8704)
    at ie (file:///Applications/Bruno.app/Contents/Resources/app.asar/web/_next/static/chunks/main-ad5649da2d8f61d3.js:1:11344)
```

I've narrowed it down to a single endpoint, but I'm sadly unable to share this data publicly. When I change the URL to an obvious wrong one, and go for another tab other than 'Response', like 'Headers', that will work. If I then change the URL to the original URL, it'll work. When switching to the 'Response' tab, it'll be an instant white screen with the above error.

Running this API call in curl shows nothing glaringly wrong, as these are the headers and response:

```
$ curl -v -4k -H 'Authorization: Token [[REDACTED]]' 'https://[[REDACTED]]/account'
*   Trying [[REDACTED]]:443...
* Connected to [[REDACTED]] ([[REDACTED]]) port 443 (#0)
* ALPN: offers h2,http/1.1
* (304) (OUT), TLS handshake, Client hello (1):
* (304) (IN), TLS handshake, Server hello (2):
* (304) (IN), TLS handshake, Unknown (8):
* (304) (IN), TLS handshake, Certificate (11):
* (304) (IN), TLS handshake, CERT verify (15):
* (304) (IN), TLS handshake, Finished (20):
* (304) (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / AEAD-AES256-GCM-SHA384
* ALPN: server accepted h2
* Server certificate:
*  subject: CN=[[REDACTED]]
*  start date: Sep  6 01:22:24 2023 GMT
*  expire date: Dec  5 01:22:23 2023 GMT
*  issuer: C=US; O=Let's Encrypt; CN=R3
*  SSL certificate verify ok.
* using HTTP/2
* h2h3 [:method: GET]
* h2h3 [:path: /account]
* h2h3 [:scheme: https]
* h2h3 [:authority: [[REDACTED]]]
* h2h3 [user-agent: curl/7.88.1]
* h2h3 [accept: */*]
* h2h3 [authorization: Token [[REDACTED]]]
* Using Stream ID: 1 (easy handle 0x14f811e00)
> GET /account HTTP/2
> Host: [[REDACTED]]
> user-agent: curl/7.88.1
> accept: */*
> authorization: Token [[REDACTED]]
> 
< HTTP/2 403 
< date: Mon, 02 Oct 2023 07:11:42 GMT
< content-type: text/plain; charset=utf-8
< content-length: 3
< x-content-type-options: nosniff
< strict-transport-security: max-age=15724800; includeSubDomains
< x-frame-options: deny
< x-xss-protection: 1; mode=block
< referrer-policy: strict-origin-when-cross-origin
< content-security-policy: default-src 'none'
< 
{}
```

Attempting to reproduce fails on any other response like httpbin throwing a 403 is unfruitful.

**Comment by helloanoop** [comment]
@Ma-ve The issue has been fixed in the latest release `v0.17.0`

**Comment by Ma-ve** [comment]
> @Ma-ve The issue has been fixed in the latest release `v0.17.0`

Much appreciated, thanks for the swift response & action!
